File: cpp/Q1.cpp
#include <iostream>
using namespace std;
int main(){

 int num;
    cout << "Digite um numero: " << endl;
    cin >> num;
    if (num%2==0) {
        cout << "Eh par" << endl;
    }
    else {
        cout << "Eh impar" << endl;
    }

    return 0;
}

File: java/Q1.java
import java.util.Scanner;

public class Main {
	public static void main(String[] args) {
		Scanner scan = new Scanner(System.in);
		int num;
		System.out.println("Digite um numero: ");
		num = scan.nextInt();
		if(num%2==0){
		    System.out.println(num + " é par");
		}
		else{
		    System.out.println(num+ " é impar");
		}
	}
}
